### DISCUSSION TITLE
Graphql filter list of objects on modificationDate

### DISCUSSION THREAD

**Discussion by DerickVanDerSandt** [pimcore/pimcore#9362] [Q&A / Support] [answered]
Is it possible to filter a list of objects on their modificationDate which is greater or smaller than x value?
If so what would the syntax look like?

**Comment by weisswurstkanone** [comment]
see https://github.com/pimcore/data-hub/blob/1.x/doc/10_GraphQL/04_Query/10_Filtering.md

**Reply by DerickVanDerSandt** [reply]
I found the same code you are referring to on the docs. However, I could not get it to work.
Can you advise on the correct syntax?

**Comment by weisswurstkanone** [comment]
can you provide a sample query which can be reproduced on demo.pimcore.fun ?

**Reply by DerickVanDerSandt** [reply]
For example to filter as usual
`{
  getCarListing (published: true, filter: "{\"name\": {\"$like\" : \"E-Type\"}}"){
    edges {
      node {
        name
        modificationDate
      }
    }
  }
}
`
But to filter on modification date:
`{
  getCarListing (published: true, filter: "{\"o_modificationDate\": {\">\" : 1614095437}}"){
    edges {
      node {
        name
        modificationDate
      }
    }
  }
}`

**Comment by weisswurstkanone** [comment, marked as answer]
Hello @DerickVanDerSandt , according to https://pimcore.com/docs/pimcore/6.9/Development_Documentation/Web_Services/Query_Filters.html it
should be "$gt" instead of ">"

So
```
{
  getCarListing(published: true, filter: "{\"o_modificationDate\": {\"$gt\" : 1614095437}}") {
    edges {
      node {
        name
        modificationDate
      }
    }
  }
}
```
Resonse would something like

```
{
  "data": {
    "getCarListing": {
      "edges": [
        {
          "node": {
            "name": "E-Type",
            "modificationDate": 1623156148
          }
        }
      ]
    }
  }
}
```

then

**Reply by DerickVanDerSandt** [reply]
Thank you very much 😁

**Reply by weisswurstkanone** [reply]
Please tell me if it works for you. My PR still needs to be merged.

**Reply by DerickVanDerSandt** [reply]
Yes it works thank you
